[2025-07-09T21:14:18.768+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-07-09T21:14:18.788+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weekly_model_retraining.retrain_model manual__2025-07-09T21:09:02.648058+00:00 [queued]>
[2025-07-09T21:14:18.793+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weekly_model_retraining.retrain_model manual__2025-07-09T21:09:02.648058+00:00 [queued]>
[2025-07-09T21:14:18.794+0000] {taskinstance.py:2306} INFO - Starting attempt 2 of 2
[2025-07-09T21:14:18.801+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): retrain_model> on 2025-07-09 21:09:02.648058+00:00
[2025-07-09T21:14:18.811+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=300) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-07-09T21:14:18.812+0000] {standard_task_runner.py:63} INFO - Started process 303 to run task
[2025-07-09T21:14:18.811+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'weekly_model_retraining', 'retrain_model', 'manual__2025-07-09T21:09:02.648058+00:00', '--job-id', '96', '--raw', '--subdir', 'DAGS_FOLDER/retraining_dag.py', '--cfg-path', '/tmp/tmpuptv9r8j']
[2025-07-09T21:14:18.813+0000] {standard_task_runner.py:91} INFO - Job 96: Subtask retrain_model
[2025-07-09T21:14:18.824+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-07-09T21:14:18.846+0000] {task_command.py:426} INFO - Running <TaskInstance: weekly_model_retraining.retrain_model manual__2025-07-09T21:09:02.648058+00:00 [running]> on host 868708e64564
[2025-07-09T21:14:18.903+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='rajat' AIRFLOW_CTX_DAG_ID='weekly_model_retraining' AIRFLOW_CTX_TASK_ID='retrain_model' AIRFLOW_CTX_EXECUTION_DATE='2025-07-09T21:09:02.648058+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-07-09T21:09:02.648058+00:00'
[2025-07-09T21:14:18.905+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-07-09T21:14:19.012+0000] {credentials.py:567} INFO - Found credentials in environment variables.
[2025-07-09T21:14:19.375+0000] {logging_mixin.py:188} INFO - [INFO] Searching for files on S3...
[2025-07-09T21:14:19.376+0000] {logging_mixin.py:188} INFO - [INFO] Found latest timestamp: 2025-07-08-15
[2025-07-09T21:14:19.376+0000] {logging_mixin.py:188} INFO - [INFO] Starting model training...
[2025-07-09T21:14:19.406+0000] {credentials.py:567} INFO - Found credentials in environment variables.
[2025-07-09T21:14:20.740+0000] {logging_mixin.py:188} WARNING - 2025/07/09 21:14:20 WARNING mlflow.utils.git_utils: Failed to import Git (the Git executable is probably not on your PATH), so Git SHA is not available. Error: Failed to initialize: Bad git executable.
The git executable must be specified in one of the following ways:
    - be included in your $PATH
    - be set via $GIT_PYTHON_GIT_EXECUTABLE
    - explicitly set via git.refresh(<full-path-to-git-executable>)

All git commands will error until this is rectified.

This initial message can be silenced or aggravated in the future by setting the
$GIT_PYTHON_REFRESH environment variable. Use one of the following values:
    - quiet|q|silence|s|silent|none|n|0: for no message or exception
    - warn|w|warning|log|l|1: for a warning message (logging level CRITICAL, displayed by default)
    - error|e|exception|raise|r|2: for a raised exception

Example:
    export GIT_PYTHON_REFRESH=quiet
[2025-07-09T21:14:22.212+0000] {logging_mixin.py:188} WARNING - 2025/07/09 21:14:22 WARNING mlflow.models.model: `artifact_path` is deprecated. Please use `name` instead.
[2025-07-09T21:14:26.414+0000] {logging_mixin.py:188} WARNING - 2025/07/09 21:14:26 WARNING mlflow.models.model: Model logged without a signature and input example. Please set `input_example` parameter when logging the model to auto infer the model signature.
[2025-07-09T21:14:26.461+0000] {logging_mixin.py:188} WARNING - Registered model 'github-anomaly-isolation-forest' already exists. Creating a new version of this model...
[2025-07-09T21:14:26.480+0000] {logging_mixin.py:188} WARNING - 2025/07/09 21:14:26 INFO mlflow.store.model_registry.abstract_store: Waiting up to 300 seconds for model version to finish creation. Model name: github-anomaly-isolation-forest, version 7
[2025-07-09T21:14:26.481+0000] {logging_mixin.py:188} WARNING - Created version '7' of model 'github-anomaly-isolation-forest'.
[2025-07-09T21:14:27.292+0000] {logging_mixin.py:188} INFO -          actor  anomaly_score  anomaly
0   0-Luminous       0.279065    False
1    0-TeXaS-0      -0.020314     True
2    0-chrollo       0.260056    False
3  00-hidan-00       0.039763    False
4    00-uno-00       0.107387    False
[2025-07-09T21:14:27.303+0000] {logging_mixin.py:188} INFO - üèÉ View run indecisive-kit-345 at: http://mlflow-server:5000/#/experiments/1/runs/0e6ee779f4484c46957680960af9acbe
[2025-07-09T21:14:27.303+0000] {logging_mixin.py:188} INFO - üß™ View experiment at: http://mlflow-server:5000/#/experiments/1
[2025-07-09T21:14:27.323+0000] {logging_mixin.py:188} INFO - [INFO] Training completed successfully.
[2025-07-09T21:14:27.324+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-07-09T21:14:27.324+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-07-09T21:14:27.333+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=weekly_model_retraining, task_id=retrain_model, run_id=manual__2025-07-09T21:09:02.648058+00:00, execution_date=20250709T210902, start_date=20250709T211418, end_date=20250709T211427
[2025-07-09T21:14:27.387+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-07-09T21:14:27.398+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-07-09T21:14:27.399+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
